### PR TITLE
[Cosmic-swingset, spawner] Update dependencies and test cosmic-swingset

### PR DESCRIPTION
**Pull request by katelynsills**
Cosmic-swingset needed to have the new npm packages added to its package.json. (ESLint didn't catch this because cosmic-swingset isn't set up for easy linting yet). 

I also noticed that our @agoric/Nat dependencies needed to be corrected to only be v2.0.1 at this point, and spawner mistakenly had the latest version (which depends on BigInt). 

Lastly, I went through the manual tests for cosmic-swingset and everything passed, although I had to make some slight changes to the tests to do things like accommodate new names for the moola and simolean assays. 

This should be the last PR for the process of moving the files out of ERTP. 🍾

### CODE REVIEW

**Comment by warner** [comment]
> This should be the last PR for the process of moving the files out of ERTP.

Oh, you might look at the CI workflow file (`.github/actions/test-all-packages.yml`) and make sure it's running the tests for the new packages. The workflow runs each package separately (rather than doing a single top-level `yarn test`) so their results will be shown separately, for easier human parsing.

File: packages/cosmic-swingset/package.json [hunk 1]
```diff
@@ -23,7 +23,15 @@
     "@agoric/evaluate": "^2.2.0",
     "@agoric/eventual-send": "^0.5.0",
     "@agoric/harden": "0.0.4",
```
Comment (warner) [inline packages/cosmic-swingset/package.json:25]: Does cosmic-swingset depend directly on these? If they're transitive dependencies, we don't really need to name them from cosmic-swingset.

File: packages/spawner/package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
     "@agoric/harden": "^0.0.4",
     "@agoric/insist": "^0.0.1",
     "@agoric/make-promise": "^0.0.1",
-    "@agoric/nat": "^3.0.1",
+    "@agoric/nat": "^2.0.1",
```
Comment (warner) [inline packages/spawner/package.json:36]: that's weird that `yarn workspaces info` didn't catch this

Comment (warner) [reply packages/spawner/package.json:36]: ohh, maybe it's because `@agoric/Nat` isn't defined in the monorepo, it's external

**Review by warner — APPROVED** [review]
LGTM

**Comment by katelynsills** [comment]
> > This should be the last PR for the process of moving the files out of ERTP.
> 
> Oh, you might look at the CI workflow file (`.github/actions/test-all-packages.yml`) and make sure it's running the tests for the new packages. The workflow runs each package separately (rather than doing a single top-level `yarn test`) so their results will be shown separately, for easier human parsing.

Gotcha, yup, I need to do this. Ok to put in this PR? Or do you want a separate one?

File: packages/cosmic-swingset/package.json (hunk 1 above)
Comment (katelynsills) [reply packages/cosmic-swingset/package.json:25]: Hmm, what's the difference? If `lib/ag-solo/` has these dependencies (which it does), which category does that fall in?

File: packages/spawner/package.json (hunk 2 above)
Comment (katelynsills) [reply packages/spawner/package.json:36]: Yeah, I think that's why

File: packages/cosmic-swingset/package.json (hunk 1 above)
Comment (warner) [reply packages/cosmic-swingset/package.json:25]: That sounds like it qualifies. I'd say: if any code in `package-X/src/*` says `import Y`, then X depends on Y. `lib/` counts as source (`test/` would make it a `devDependency`).

(there were so many dependencies added that I thought maybe we were adding all the new packages just to be conservative; also at some point in the past, the transitive-crawler wasn't working right and we had to include the whole tree as a workaround, but I think that should be fixed now)

**Comment by warner** [comment]
> Gotcha, yup, I need to do this. Ok to put in this PR? Or do you want a separate one?

separate PR is good

File: packages/cosmic-swingset/package.json (hunk 1 above)
Comment (katelynsills) [reply packages/cosmic-swingset/package.json:25]: Ok, great!

**Pull request merged**